src/main/java/org/apache/maven/plugins/shade/mojo/ShadeMojo.java

```diff
@@ -285,6 +286,14 @@
     @Parameter( defaultValue = "false" )
     private boolean generateUniqueDependencyReducedPom;
 
+    /**
+     * Do we put the dependency reduced pom in the jar instead of the jar file provided by the project.
+     *
+     * @since 3.2.5
```

Comment:
3.3.0

Reply:
Apparently more went wrong at my end.
Fixing all of those.
Re checking all historical feedback.